### PR TITLE
CC-5047: connect jdbc url field should have example text

### PR DESCRIPTION
**Pull request by dosvath**

### CODE REVIEW

**Comment by gharris1727** [comment]
@dosvath Can you also regenerate this section of the docs and open a PR to verify that the formatting is correct?

**Comment by dosvath** [comment]
[clabot:check]

**Comment by dosvath** [comment]
Docs PR: https://github.com/confluentinc/docs/pull/3663

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java [hunk 1]
```diff
@@ -51,7 +51,12 @@
   );
 
   public static final String CONNECTION_URL = "connection.url";
-  private static final String CONNECTION_URL_DOC = "JDBC connection URL.";
+  private static final String CONNECTION_URL_DOC =
+      "JDBC connection URL.\n"
+          + "For example, ``jdbc:oracle:thin:@localhost:1521:orclpdb1``, "
```
Comment (gharris1727) [inline src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java]:
```suggestion
          + "For example: ``jdbc:oracle:thin:@localhost:1521:orclpdb1``, "
```

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java [hunk 2]
```diff
@@ -51,7 +51,12 @@
...
+          + "``jdbc:mysql://localhost/high_performance_java_persistence``, "
```
Comment (gharris1727) [inline src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java]: Make these shorter so they don't wrap in the documentation if possible.
```suggestion
          + "``jdbc:mysql://localhost/db_name``, "
```

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java [hunk 3]
```diff
@@ -51,7 +51,12 @@
...
+          + "``jdbc:sqlserver://localhost;instance=SQLEXPRESS;"
+          + "databaseName=high_performance_java_persistence``";
```
Comment (gharris1727) [inline src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java]:
```suggestion
          + "databaseName=db_name``";
```

**Pull request merged**